packages/react-bootstrap-table2-filter/src/filter.js

```diff
@@ -16,7 +16,8 @@ export const filterByText = _ => (
     if (comparator === EQ) {
       return cellStr === filterVal;
     }
-    return cellStr.indexOf(filterVal) > -1;
+    const re = RegExp(filterVal, 'i');
```

Comment:
is that danger? for example `\test\` or `\\test\\`, maybe throw some err?

Reply:
Like if the `filterVal` is `\test\`?
I’ll do some testing to see if that can cause an error.